packages/runtime/src/runtimes/react/components/ElementData.tsx

```diff
@@ -15,16 +15,18 @@ export const ElementData = memo(
     ref: Ref<unknown>,
   ): JSX.Element {
     const Component = useComponent(elementData.type)
+    const { builtinSuspense } = useComponentMeta(elementData.type) ?? {}
 
     if (Component == null) {
       console.warn(`Unknown component '${elementData.type}'`, { elementData })
       return <FallbackComponent ref={ref as Ref<HTMLDivElement>} text="Component not found" />
     }
 
     const forwardRef = canAcceptRef(Component)
+    const SuspenseOrFragment = builtinSuspense ?? true ? Suspense : Fragment
```

Comment:
Continue to wrap every component with `<Suspense>` by default, opt out if `builtinSuspense` is false.